file_feature2.txt

```diff
@@ -0,0 +1 @@
+this file was created in feature2
```

Comment:
what do you think about this line nabilstudies?

Reply:
this one is good too but it might be better, good job